esp/services/ws_cloud/ws_cloudService.cpp

```diff
@@ -149,10 +203,39 @@ bool CWsCloudEx::onGetServices(IEspContext& context, IEspGetServicesRequest& req
     return true;
 }
 
+// Indented for reference and ease of reading
```

Comment:
I can remove this comment if you think it is undesirable to keep in.

Reply:
Any "real" reason to not just keep the indented version?